### PR TITLE
Add `signbit` operation

### PR DESCRIPTION
**Pull request by rdspring1**
Fixes https://github.com/NVIDIA/Fuser/issues/314

### CODE REVIEW

File: python_tests/test_python_frontend.py [hunk 1]
```diff
@@ -1312,6 +1312,24 @@ def fusion_func(fd: FusionDefinition):
...
+    def test_signbit(self):
+        inputs = [
+            torch.randn(3, 4, 5, device="cuda", dtype=torch.float32),
+            torch.randn(3, 4, 5, device="cuda", dtype=torch.float32),
+        ]
+
+        def fusion_func(fd: FusionDefinition):
+            t0 = fd.from_pytorch(inputs[0])
+            t1 = fd.from_pytorch(inputs[1])
+            t2 = fd.ops.where(fd.ops.signbit(t0), -abs(t1), abs(t1))
```
Comment (csarofeen) [inline python_tests/test_python_frontend.py:1324]: Why compare with where instead of just direct signbit? Are you just making sure it fuses correctly?

**Review by csarofeen — APPROVED** [review]
Just a question on the test, otherwise LGTM.

File: python_tests/test_python_frontend.py (hunk 1 above)
Comment (rdspring1) [reply python_tests/test_python_frontend.py:1324]: No particular reason. This is how `copysign` is defined in thunder. 

https://github.com/Lightning-AI/lightning-thunder/blob/main/thunder/clang/__init__.py#L846

**Review by kevinstephano — APPROVED** [review]

**Comment by rdspring1** [comment]
!build

**Pull request merged**